src/utils/wallet.js

```diff
@@ -451,237 +421,25 @@ const wallet = {
-      }
-    });
-
-    wallet.on('new-tx', async (tx) => {
-      let message = '';
-      if (helpers.isBlock(tx)) {
-        message = 'You\'ve found a new block! Click to open it.';
-      } else {
-        message = 'You\'ve received a new transaction! Click to open it.'
-      }
-
-      const notification = this.sendNotification(message);
-
-      // Set the notification click, in case we have sent one
-      if (notification !== undefined) {
-        notification.onclick = () => {
-          routerHistory.push(`/transaction/${tx.tx_id}/`);
-        }
-      }
-
-      this.fetchNewTxTokenBalance(wallet, tx).then(async (updatedBalanceMap) => {
-        if (updatedBalanceMap) {
-          this.handlePartialUpdate(wallet, updatedBalanceMap);
-
-          // If we are here, we have already fetched new addresses, so it's safe to
-          // call getCurrentAddress to make sure we display a new address to the user
-          const currentAddress = wallet.getCurrentAddress();
-          this.updateSharedAddressRedux(currentAddress.address, currentAddress.index);
-        }
-      });
-    });
-
-    wallet.on('update-tx', async (tx) => {
-      // `update-tx` event is not yet being emitted from the wallet-service facade as it is not implemented,
-      // we should ignore this message for now to prevent errors when we start emitting it in the future
-      if (useWalletService) {
-        return;
-      }
-      const balances = await wallet.getTxBalance(tx, { includeAuthorities: true });
-      const updatedBalanceMap = await this.fetchNewTxTokenBalance(wallet, tx);
-      store.dispatch(updateTx(tx, updatedBalanceMap, balances));
-    });
-
-    try {
-      const serverInfo = await wallet.start({ pinCode: pin, password });
-
-      this.setConnectionEvents(connection, wallet);
-    } catch(e) {
-      if (useWalletService) {
-        // Wallet Service start wallet will fail if the status returned from
-        // the service is 'error' or if the start wallet request failed.
-        // We should fallback to the old facade by storing the flag to ignore
-        // the feature flag
-        await featureFlags.ignoreWalletServiceFlag();
-
-        // Restart the page, ignoring cache to make sure events are reset
-        walletHelpers.reloadElectron();
-      }
-    }
-
-    featureFlags.on('wallet-service-enabled', (newUseWalletService) => {
-      // We should only force reset the bundle if the user was on
-      // the wallet service facade and the newflag sends him to
-      // the old facade
-      if (useWalletService && useWalletService !== newUseWalletService) {
-        walletHelpers.reloadElectron();
-      }
-    });
-  },
-
-  setConnectionEvents(connection, wallet) {
-    connection.on('best-block-update', (height) => {
-      // HTR balance might have updated because of new height
-      // and some block HTR might have unlocked
-      this.fetchNewHTRBalance(wallet).then((htrUpdatedBalance) => {
```

Comment:
Can we remove this `fetchNewHTRBalance` method?

Reply:
Yes we can, done in 54e0653e27af8005ff919e0825c78d3110e68015